backend/index.js

```diff
@@ -3,28 +3,29 @@ const app = express()
 const axios = require('axios');
 app.use(express.json());
 
+app.get('/addAnimal', function(req,res){
 
-/*
-    I will leave this here as an example that you can use
-    when building your APIs. I suggest that you try to understand
-    what each line is doing. Reach out if you have any questions.
-*/
-app.get('/exampleApi', function (req, res) {
+    const body = {
+        "id":1,
+        "record": {
+            "name": "Brownie",
+            "type": "Labrador Retriever",
+            "sex": "Male",
+            "shelter_name": "Austin Pets Alive",
+            "adoption_url": "https://www.austinpetsalive.org/adopt/dogs/apa-a-56615",
+            "image": "https://www.austinpetsalive.org/assets/animals/_largeThumb/20190606003943.jpeg"
+        }
+    }
```

Comment:
Rosey remember that this JSON object should be coming in from the body of the request from postman.